PowerEditor/installer/APIs/php.xml

```diff
@@ -12221,10 +12221,10 @@
 			</Overload>
 		</KeyWord>
 		<KeyWord name="stream_get_contents" func="yes">
-			<Overload retVal="string">
-				<Param name="resource handle"/>
-				<Param name="[int maxlength=-1"/>
-				<Param name="[int offset=0]]"/>
+			<Overload retVal="string|false">
+				<Param name="resource stream"/>
+				<Param name="[int length=-1"/>

```

Comment:
rather `int $length = null` ?
ref: https://www.php.net/manual/en/function.stream-get-contents.php

Reply:
@vlakoff 
Thank you for your reply.

Reply:
@vlakoff 
Thank you for your reply.